<launch>
  <node pkg="pr2_power_board" type="power_node" name="power_node" />
  <!-- Disable power board -->
  <node pkg="qualification" type="power_board_cmd.py" name="power_cmd"
        args="--srv=shutdown_done  --cmd=disable" />
</launch>
